usagov/vote-gov-drupal
# VOTE-2888: Configure in-page navigation

## Jira ticket

[VOTE-2888](https://cm-jira.usa.gov/browse/VOTE-2888)

## Description

Adds aside for in-page navigation and configures it to only show on State pages that are NOT not-needed registration types.

## Deployment and testing

### QA/Testing instructions

1. lando retune
2. Add content for a state of each registration type
3. Check the in-page navigation is pulling the correct headers/subheaders
4. Test it's functionality when clicking through the navigation
5. Check that it does not show for not-needed state, only online and by-mail

## Checklist for the Developer




## Checklist for the Peer Reviewers
